src/main/resources/templates/gardens/list.html

```diff
@@ -8,36 +8,38 @@
             display: none;
         }
     </style>
+    <script type="text/javascript" th:src="@{/script.js}"></script>
 </head>
-<body>
+<body class="gardens">
+
 <div th:replace="fragments :: page-header"></div>
-<div class="container body-content">
-    <h2 th:text="${title}">Gardens</h2>
 
-<!--    Below div iterates through the collection and makes a row for every elements with an index divisible by 3 e.g. 0, 3, 6
-        adding rowStatus uses a thymeleaf mechanism to create a variable with an index property that holds the current iteration index -->
-    <div th:each="garden,rowStatus : ${gardens}" class="row grid-row" th:if="${rowStatus.index % 3} == 0">
-<!--    Below div iterates through the collection again with a new status variable and adds gardens to the row if this index is less than
-        the row index + 3 and the index is >= -->
-        <div th:each="garden,gardenStatus : ${gardens}" class="col-sm"
-             th:if="${gardenStatus.index < rowStatus.index + 3 and gardenStatus.index >= rowStatus.index}">
-            <img th:src="@{${'/user/garden/list/' + garden.id}}" style="width: 75px;" alt="Beautiful Garden"/>
-            <p th:text="${garden.name}"></p>
-            <p th:text="${garden.address}"></p>
-            <p th:text="${'Owner: ' + garden.owner.name}"></p>
-            <p th:text="${'Description: ' + garden.description}"></p>
-            <a th:href="@{'/user/view/' + ${garden.id}}">More Information</a>
+<h2 class="garden-title">Water Gardens</h2>
```

Comment:
If you wrap line 17-42 in a `<div class="container">` it will help with alignment and consistency of content width.

Reply:
I just tried the container, but it looks like it only shows two gardens across instead of three.  It didn't seem to change anything else either.  Did you try it and see something different?  thx.

Reply:
Interesting, it may just be issues between our screen resolutions, or using Safari vs Chrome. I'm not certain. 
Whenever I put the content in a div container I get three across. Without the container the content stretches the full width of the page, I get 5 across and 1 below since I have 6 gardens in my database.

Reply:
Just saw the slack messages, sorry about that!